lib/ts/recipe/emailpassword/api/implementation.ts

```diff
@@ -63,6 +63,9 @@ export default function getAPIImplementation(): APIInterface {
                   status: "OK";
               }
             | { status: "PASSWORD_RESET_NOT_ALLOWED"; reason: string }
+            | {
+                  status: "USER_BANNED_ERROR";
+              }
```

Comment:
the API should not output this, as we already have PASSWORD_RESET_NOT_ALLOWED as a possible output.